### PR TITLE
Add placeholders for dataset overviews

### PR DESCRIPTION
**Pull request by znichollscr**
## Checklist

Please confirm that this pull request has done the following:

- [x] Documentation added (where applicable)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

**Comment by znichollscr** [comment]
@durack1 @vnaik60 what do you reckon?

**Comment by znichollscr** [comment]
Relevant pages for doc reviews:

- https://input4mips-controlled-vocabularies-cvs--146.org.readthedocs.build/en/146/usage-data-user/
- https://input4mips-controlled-vocabularies-cvs--146.org.readthedocs.build/en/146/dataset-overviews/

**Review by durack1 — APPROVED** [review]
Looks like a good start

File: docs/database-views/input4MIPs_delivery-summary_CMIP6Plus.html [hunk 1]
```diff
@@ -78,48 +78,55 @@ <h4>
...
+      <td>11</td>
       <td>Aerosol optical properties/MACv2-SP</td>
       <td>TBD</td>
       <td>Depends on 1, test dataset being produced in the meantime</td>
       <td>Expected: ~November 2024; Expected a month after dependent datasets</td>
     </tr>
     <tr>
-      <td>11</td>
+      <td>12</td>
       <td>Population</td>
```
Comment (vnaik60) [inline docs/database-views/input4MIPs_delivery-summary_CMIP6Plus.html:130]: Population Density

File: docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Aerosol optical properties/MACv2-SP
```
Comment (vnaik60) [inline docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md:1]: Should we also include contact info on the data? Linking to data provider info.

File: docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Two variables can be calculated from the data:
+
+1. aerosol optical properties
+1. an adjustment to cloud-droplet population densities
+
+### Uncertainty
+
+[TBD, recommendations on parameter ranges?]
+
+## Differences from CMIP6
```
Comment (vnaik60) [inline docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md:40]: Differences from CMIP6 or any other dataset previously available. For example, we did not have CMIP6 recommended volcanic SO2 emissions but there is a dataset that existed and was used by at least modeling group.

File: docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Aerosol optical properties/MACv2-SP
+
+## Summary
```
Comment (vnaik60) [inline docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md:3]: Should we include something on Revision History

**Review by vnaik60 — CHANGES_REQUESTED** [review]

File: docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md (hunk 2 above)
Comment (znichollscr) [reply docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md:1]: Yes good point. I'm a little bit hesitant about duplicating information and having it become out of date, but on this way the duplication is the right choice I think.

File: docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md (hunk 3 above)
Comment (znichollscr) [reply docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md:40]: Ah ok nice, good point.

File: docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md (hunk 4 above)
Comment (znichollscr) [reply docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md:3]: Yes probably. I think we should capture that in the dataset view of our database so I wouldn't duplicate that here, but having a link to that resource is a good idea.

**Comment by durack1** [comment]
Couldn't hurt to point out that we also have the CMIP6-era datasets available alongside the CMIP6Plus-era datasets.. Makes me think about how relevant it is to provide the historical context of these data and their evolution

File: docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md (hunk 4 above)
Comment (znichollscr) [reply docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md]: Ok I've thought about it a bit more. It's a really good idea, but I'm going to need to do an hour of playing around to make it maintainable and easy I think. Let's see if I find that hour tomorrow or Friday.

**Pull request closed without merging**